### ISSUE TITLE
some questions about pertained model 

### ISSUE THREAD

**Issue by bestfleer** [JiahuiYu/slimmable_networks#8]
Hello, which dataset you used to pretained model? Imagine 1K?   When I load_state_dict your pertained model, it has low accuracy on ImageNet 1k. the version of ImageNet 1k I used is ILSVRC2012. Really appreciate your work and your help.

**Comment by bestfleer** [comment]
238.3s	val	1.0	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
238.3s	val	0.75	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
238.4s	val	0.5	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
238.4s	val	0.25	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844

**Comment by zh583007354** [comment]
> 238.3s	val	1.0	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
> 238.3s	val	0.75	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
> 238.4s	val	0.5	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844
> 238.4s	val	0.25	0/100: loss: 14.389, top1_error: 0.987, top5_error: 0.844

hi. My result on s_mobilenet_v1 with imagenet1k is:
> 238.3s	val	1.0	-1/480: loss: 1.246, top1_error: 0.298, top5_error: 0.107
> 238.3s	val	0.75	-1/480: loss: 1.320, top1_error: 0.318, top5_error: 0.118
> 238.4s	val	0.5	-1/480: loss: 1.534, top1_error: 0.366, top5_error: 0.150
> 238.4s	val	0.25	-1/480: loss: 2.111, top1_error: 0.481, top5_error: 0.237

**Comment by JiahuiYu** [comment]
@zh583007354 Thanks for your help to @bestfleer !
